api/api.yaml

```diff
@@ -10,6 +10,32 @@ paths:
   /facets:
     get:
       description: Returns facets.
+      parameters:
+        - name: filter
+          # Ideally this query parameter would use an object. However, OpenAPI
```

Comment:
This is all context behind this PR which can go into the PR description/comments instead of cluttering the API definition itself. Additionally, could you file an issue for updating to 3.0 and then changing this to an object?

Reply:
I prefer to have Issues for short-term things (next month or so); the [Issues tab](https://github.com/DataBiosphere/data-explorer/issues) is getting kind of hard to read. For things that might never happen, or could happen in next year or two, I prefer TODO. WDYT?

I guess an alternative is to use labels and label every Issue as short-term or long-term, but it's simpler just to only file issues for short-term.

Reply:
If you dislike github issues we could reconsider our issue tracking service. Regardless, I don't think paragraphs of context belong in the code base.

I generally follow the Google style guide for TODOs. That is, use them for temporary or short term code and always assign them to either a person or a bug. In this case, if you wanted to make this a TODO I would file the github issue and document it here as `TODO (#XX): Change to object on swagger update to 3.0`

Reply:
Done. In retrospect I agree with you. I left one small comment because it's strange to have a string like "Gender=female"; normally things would be broken up into objects.